modules/menus/menu_actor_spawn/include/actor_spawn_menu.h

```diff
@@ -0,0 +1,51 @@
+#include "menu.h"
+
+#define MAX_ACTORS 10
+
+/**
+ * @struct procBinData
+ * @brief Structure for entries in res/proc_info/procs.bin
+ */
+struct procBinData {
+    s16 procId;
+    char procName[30];
+} __attribute__((aligned(32)));
```

Comment:
this struct already exists for the actor view menu right? we probably shouldn't duplicate it